tests/code/test_listOfTerms.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from app.controllers.admin.routes import createEventPage
+from app.models.term import Term
+
+@pytest.mark.integration
+def test_validateListOfTerms():
+    currentTermid = 3
+    termQuery = (Term.select()
+                      .where((Term.year <= 2023))
+                      .where(Term.id >= currentTermid))
+

```

Comment:
You're not actually calling `selectFutureTerms()` in this test. You're doing the query yourself.